.circleci/config.yml

```diff
@@ -461,6 +461,21 @@ workflows:
           requires:
             - Install NPM packages
 
+      - build-and-save-docker-image:
+          name: Build base image (Ubuntu)
+          requires:
+            - Install NPM packages
+          dockerfile: dockerfiles/base/Dockerfile
+          nodejs_cycle: "20"
+          project_name: broker
+
+      - build-and-save-docker-ubi-image:
+          name: Build base image (RHEL)
+          requires:
+            - Install NPM packages
+          dockerfile: dockerfiles/base/Dockerfile.ubi
+          project_name: broker-rhel-ubi
+
```

Comment:
nit: we should require these to pass in the following `release` job

Reply:
great point

Reply:
makes sense, good catch.

Reply:
done with dffbc9b